client/assets/style/style.css

```diff
@@ -1,20 +1,49 @@
-@import url(https://fonts.googleapis.com/css?family=Tangerine);
-@import url(https://fonts.googleapis.com/css?family=Playfair+Display);
+@import url(https://fonts.googleapis.com/css?family=Parisienne);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400italic,700);
 
 .title h2 {
   padding-left: 20px;
   font-size: 4em;
-  font-family: 'Tangerine', cursive;
-  color: #848482;
+  font-family: 'Parisienne', cursive;
+  color: #3e3e3e;
 }
 
 html, body {
   height:100%;
 }
 
-.main-app {
-  font-family: 'Playfair Display', serif;
-  font-size: 16px;
+a {
+  color: #f3785b;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #f8b4a4;
+  text-decoration: none;
+}
+
+.h2, h2 {
```

Comment:
i don't think we have any .h2 classes so h2 should just be good enough, same for h3 and h4